src/modules/common/components/FailureSwitcher.js

```diff
@@ -58,8 +57,6 @@ export class FailureSwitcher extends React.Component<PropsType> {
     const error = this.props.error
     if (error instanceof ContentNotFoundError) {
       return FailureSwitcher.renderContentNotFoundComponent(error)
-    } else if (error instanceof LanguageNotFoundError) {
```

Comment:
Is that handled somewhere else now?

Reply:
Currently also in the Switcher, because the failure can only occur there.